impeller/toolkit/android/hardware_buffer.cc

```diff
@@ -121,4 +134,37 @@ std::optional<AHardwareBuffer_Desc> HardwareBuffer::Describe(
   return desc;
 }
 
+void* HardwareBuffer::Lock(CPUAccessType type) const {
```

Comment:
Are we using Lock/unlock or was this only for testing?

Reply:
This was just for testing. But I'm inclined to leave it in there since it is handy when you need it and get LTOd away when you don't. But your call on if this should be backed out.

Reply:
I think its fine to leave them